ui/src/treeDataProvider/SubscriptionNode.ts

```diff
@@ -12,7 +12,7 @@ import { AzureSubscription } from '../azure-account.api';
 import { AzureParentNode } from './AzureParentNode';
 
 export class SubscriptionNode extends AzureParentNode {
-    public static readonly contextValue: string = 'azureSubscription';
+    public static readonly contextValue: string = 'azureextensionui.azureSubscription';
```

Comment:
This should be a minor version bump since it technically breaks functionality in extensions. For example:
* app service: https://github.com/Microsoft/vscode-azureappservice/blob/master/package.json#L262
* functions: https://github.com/Microsoft/vscode-azurefunctions/blob/master/package.json#L227

Reply:
@EricJizbaMSFT I'll take it then that you won't as republish 6.0.1.  I'll go ahead and update the version.

Reply:
I can't republish 0.6.1. You have to bump the version no matter what and I would prefer bumping the minor :)

Reply:
Realized what you meant after I wrote that. :-)